package.json

```diff
@@ -25,13 +25,14 @@
     "@babel/preset-react": "^7.22.15",
     "@eslint/js": "^9.23.0",
     "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.1",
     "autoprefixer": "^10.4.16",
     "babel-loader": "^10.0.0",
     "css-loader": "^7.0.0",
     "cypress": "^14.0.1",
     "cypress-axe": "^1.5.0",
-    "eslint": "~9.28.0",
+    "eslint": "~9.27.0",
```

Comment:
Was this rolled back intentionally? Otherwise you might need to rebase before merging

Reply:
Hm. This is weird. When I rebase it refuses to change. And this change didn't even update the package.lock